apb-base/Dockerfile

```diff
@@ -9,11 +9,13 @@ ENV USER_NAME=apb \
 ENV HOME=${BASE_DIR}
 
 RUN mkdir -p /root/.kube /usr/share/ansible/openshift \
+             /etc/ansible-service-broker \
              /etc/ansible /opt/ansible \
              ${BASE_DIR} ${BASE_DIR}/etc \
              ${BASE_DIR}/.kube ${BASE_DIR}/.ansible/tmp && \
              useradd -u ${USER_UID} -r -g 0 -M -d ${BASE_DIR} -b ${BASE_DIR} -s /sbin/nologin -c "apb user" ${USER_NAME} && \
              chown -R ${USER_NAME}:0 /opt/{ansible,apb} && \
+             chown -R ${USER_NAME}:0 /etc/ansible-service-broker && \
```

Comment:
I actually find it weird to see `ansible-service-broker` in the base apb image. Maybe it's not a bad thing, but I find it weird to see it leaking in. Especially in `/etc/` which I think of a config directory